indico/migrations/versions/20200409_1711_7f56252c73ab_add_map_url_to_events_table.py

```diff
@@ -0,0 +1,24 @@
+"""Add map_url to events table
+
+Revision ID: 7f56252c73ab
+Revises: 933665578547
+Create Date: 2020-04-09 17:11:13.685047
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '7f56252c73ab'
+down_revision = '933665578547'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('events', sa.Column('_map_url', sa.String(), nullable=True, server_default=''), schema='events')
```

Comment:
please don't change the name here; you can pass `'map_url'` as the first posarg to `db.Column()` to set the column name separately from the